src/components/Slider.react.js

```diff
@@ -133,6 +138,49 @@ Slider.propTypes = {
      */
     max: PropTypes.number,
 
+    tooltip: PropTypes.oneOfType([
+        /**
+         * Determines whether tooltips should always be visible
+         * (as opposed to the default, visible on hover)
+         */
+        PropTypes.exact({
+            visible: PropTypes.bool.isRequired,
+        }),
+
+        /**
+         * Determines the position of tooltips
+         * See https://github.com/react-component/tooltip#api
+         * top/bottom{*} sets the _origin_ of the tooltip, so e.g. `topLeft` will
+         * in reality appear to be on the top right of the handle
+         */
+        PropTypes.exact({
+            position: PropTypes.oneOf([
+                'left',
+                'right',
+                'top',
+                'bottom',
+                'topLeft',
+                'topRight',
+                'bottomLeft',
+                'bottomRight',
+            ]).isRequired,
+        }),
+
+        PropTypes.exact({
+            visible: PropTypes.oneOf(['visible', 'hover']),
```

Comment:
elsewhere `visible` is a `bool` - which is right?

Reply:
Good catch, it should be a bool in both `Slider` and `RangeSlider`.